Makefile

```diff
@@ -283,10 +288,13 @@ manifest-annotate-%:
 	done
 
 manifest-name:
-	@echo "manifest: $(IMAGE_ROOT):$(VERSION)"
+	@echo "manifest: $(IMAGE):$(VERSION)"
 
 push: .push-$(ARCH)-$(VERSION) push-name
 .push-$(ARCH)-$(VERSION): .container-$(ARCH)-$(VERSION)
+ifneq ($(REGISTRY_REFIX),)
+	@docker tag $(IMAGE):$(ARCH)-$(VERSION) $(REGISTRY)/$(IMAGE):$(ARCH)-$(VERSION)
```

Comment:
Maybe for consistency we should replace all occurrences of `$(REGISTRY)/$(IMAGE)` with `$(MANIFEST_IMAGE)` or `$(FULLY_QUALIFIED_IMAGE)`?

Reply:
Done, see comment bellow about the code change as I didn't test on docker hub.